packages/devextreme/testing/tests/DevExpress.ui.widgets/tabs.tests.js

```diff
@@ -506,7 +506,7 @@ QUnit.module('Tab select action', () => {
     });
 
     QUnit.test('focusedElement must be changed after changing the selectedIndex', function(assert) {
```

Comment:
We can use script iteration with a change in selectedIndex, selectedItem(s) to avoid duplication test scenarios.

Reply:
No, we can't. There are too many differences in configurations for selectedIndex, selectedItem and selectedItems.

Reply:
Okay, if that's more convenient, let's not merge then